lib/at_cmd_parser/at_cmd_parser.c

```diff
@@ -61,6 +63,21 @@ static int at_parse_detect_type(const char **str, int index)
 		 * notification ID, (eg +CEREG:)
 		 */
 		set_new_state(NOTIFICATION);
+
+		if (!strncmp(tmpstr, "%HWVERSION", 10)) {
+				setTypeString = true;
+		}
+		if (!strncmp(tmpstr, "%SHORTSWVER", 11)) {
```

Comment:
Would it be more efficient for the following ifs be ``esle if``? Looks like they can not occur at the same time.

Reply:
Done